plugins/scanners/dos/src/main/kotlin/DosScanner.kt

```diff
@@ -160,12 +161,12 @@ class DosScanner internal constructor(
     }
 
     internal suspend fun runBackendScan(
-        purls: List<String>,
+        dosPackages: List<PackageInfo>,
```

Comment:
I propose to avoid the `dos` prefix, for consistency with other places and to keep it simple.